app/src/export.js

```diff
@@ -0,0 +1,204 @@
+    },[params.sid]);
+
+    const SessionSection = () => {
+        console.log(comprehendDict)
+        const Session = comprehendDict.Session
+
+        return (
+            <div className={s.session}>
+                <main>
+                    
+                    <Table className={s.ses} bordered striped small>
+                        <thead>
+                            <tr>
+                                <td colSpan='5'><h3>Session Overview</h3></td>
+                            </tr>
+                            <tr>
+                                <th>Session Id</th>
+                                <td className={s.spec}>{Session.sessionId}</td>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            
+                            <tr>
+                                <th>Patient Id </th>
+                                <td>{Session.patientId}</td>
+                            </tr>
+                            <tr>
+                                <th>Health Care Professional Id</th>
+                                <td>{Session.healthCareProfessionalId}</td>
+                            </tr>
+                            <tr>
+                                <th>Timestamp Start </th>
+                                <td>{Session.timeStampStart}</td>
+                            </tr>
+                            <tr>
+                                <th>Timestamp End</th>
+                                <td>{Session.timeStampEnd}</td>
+                            </tr>
+                        </tbody>
+                    </Table>
+                </main>
+            </div>
+        )
+
+    }   
+
+    const TranscribeSection = () => (
+        <div className={s.transcribe}> 
+            <h2>Session Transcription</h2>
+            {transcribeText.map( (text) => 
+                { return text ? <p>{text+'.'}</p> : null}
+            )}
+        </div>
+    )
+
+    const ComprehendSection = () => {
+        const Medication = comprehendDict.Medication
+        const RxNorm = comprehendDict.RxNorm
+        const MedicationRxNorm = comprehendDict.MedicationRxNorm
+        const MedicalCondition = comprehendDict.MedicalCondition
+        const ICD10CMConcept = comprehendDict.ICD10CMConcept
+        const MedicalConditionICD10CMConcept = comprehendDict.MedicalConditionICD10CMConcept
+        const TestTreatmentProcedures = comprehendDict.TestTreatmentProcedures
+    
+        return (
+            <div className={s.comprehend}>
+                <main>
+                
+                        <Table className={s.tests} bordered striped >
+                            <thead>
+                                <tr>
+                                    <td colSpan='3'><h3>Tests, Treatments, Procedures </h3></td>
+                                </tr>
+                                <tr>
+                                    <th>#</th>
```

Comment:
Just Curious, Why do we have this table header?

Reply:
It is for the table listing, we can change it later.